tests/generic/nexusmutual_product_information_column_values.sql

```diff
@@ -0,0 +1,24 @@
+{% test nexusmutual_product_information_column_values(model, column_name, nexusmutual_ethereum_product_information_seed) %}
+
+    with unit_test as
+    (
+        select
+            case
+                when m.{{ column_name }} = seed.{{ column_name }}
+                    then True
+                else False
+            end as generic_column_test
+        from {{ model }} m
+        join {{ nexusmutual_ethereum_product_information_seed }} seed
+            on  m.product_contract_address = seed.product_contract_address
+            and m.product_name = seed.product_name
+            and m.product_type = seed.product_type
+            and m.date_added = seed.date_added
+            and m.syndicate = seed.syndicate
```

Comment:
Since this the current test joins all columns in the seed, the test checking will not work as expected.

I have modified the test part in the schema file, by using `product_contract_address` as unique key to join, and check [`product_name`, `product_type`, `syndicate`] columns.

I will remove [tests/generic/nexusmutual_product_information_column_values.sql](https://github.com/duneanalytics/spellbook/pull/2485/files#diff-692cca1903c643ee9de670e92613d3978566103dd3d4c68fad161401fead695f) in the next commit as we can use the `check_seed` generic test.